libdieharder/rng_file_input.c

```diff
@@ -170,7 +170,7 @@ static unsigned long int file_input_get (void *vstate)
    state->rptr++;
    state->rtot++;
    if(verbose){
-     fprintf(stdout,"# file_input() %lu: %lu/%lu -> %u\n", state->rtot, state->rptr,state->flen,(uint)iret);
```

Comment:
Note that the original `off_t` is an alias for the signed type `long int`.  Furthermore, one can never know what `off_t` actually is. Could be any signed 64-bit-integer, e.g., `long int` or `long long int` on Linux on x64, which are distinct types with distinct format strings.